code/controllers/subsystem/garbage.dm

```diff
@@ -145,12 +145,8 @@ SUBSYSTEM_DEF(garbage)
 			pass_counts[i] = 0
 			fail_counts[i] = 0
 
-#ifdef EXPERIMENT_515_QDEL_HARD_REFERENCE
 // 1 from the hard reference in the queue, and 1 from the variable used before this
-#define IS_DELETED(datum, _) (refcount(##datum) == 2)
-#else
-#define IS_DELETED(datum, gcd_at_time) (isnull(##datum) || ##datum.gc_destroyed != gcd_at_time)
-#endif
+#define IS_DELETED(datum) (refcount(##datum) == 2)
```

Comment:
feels better to inline this and move the comment in the actual code

Reply:
sure